utils/preflight_check.py

```diff
@@ -433,30 +433,30 @@ def main(
     parameters = get_taskgraph_parameters()
     run_taskgraph(parsed_args.config, get_taskgraph_parameters())
 
-    match Choices[parsed_args.only] if parsed_args.only else None:
```

Comment:
Question: Why is this changed like this?

Reply:
I ran into issues with this under Python 3.10. The change is trivial but allows backward compatibility in case other people will run it under < 3.11.